getconf/base.py

```diff
@@ -169,6 +170,19 @@ def gettimedelta(self, key, default='0d', doc=''):
         else:
             return datetime.timedelta(**{unit: value})
 
+    def getpath(self, key, default=Path(), doc=''):
```

Comment:
Let's make it obvious, then. :)

```suggestion
    def getpath(self, key, default=Path('.'), doc=''):
```

(By the way, I had a doubt whether `Path` objects are immutable. [They are](https://docs.python.org/3/library/pathlib.html#general-properties). So they are fine as a default argument value.)